### PR TITLE
Request for code review

### PR DESCRIPTION
**Pull request by patriciachrysy**
Dear TSE,
- Created the bin and lib folders
- Created the main.rb file inside the bin folder
- added the gitkeep file to lib folder
- Made the main.rb file executable
Waiting for your review

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/home/manezeu/.rbenv/shims/ruby
```
Comment (MauricioRobayo) [inline bin/main.rb]: This line should be `#!/usr/bin/env ruby`. You can test it by executing the file without a program to run it on the command line:

```
$ ./bin/main.rb 
hello world!
```

The way you have will work on `manezeu` computer as it is using that username, and will only work if ruby was installed with `rbenv`. We want it to work on every *nix with Ruby installed, so  `#!/usr/bin/env ruby` will pick the Ruby executable from the PATH.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Well done!

It looks great and is coming into place, but we need to fix some issues before continuing:

* [ ] Fix the [`shebang`](https://en.wikipedia.org/wiki/Shebang_(Unix)) line on your `./bin/main.rb`. It should be `#!/usr/bin/env ruby`.
* [ ] Remove last line on `.stickler.yml`. Seems like something weird slipped into it and that's making Stickler fail.

Status: __Request changes__.

**Comment by patriciachrysy** [comment]
Changes applied according to review, please check again.

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good job implementing the suggested chnages. :relaxed: 

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**